### DISCUSSION TITLE
How can I fetch forge notifications on magit-pull?

### DISCUSSION THREAD

**Discussion by artpelling** [magit/forge#459] [Support] [answered]
Is there way I can configure magit and forge to automatically fetch forge notifications and topics when envoking magit pull?

**Comment by tarsius** [comment, marked as answer]
Unfortunately not. Fetching notifications is terribly inefficient and blocks emacs longer than other api fetches, so I have opted to make it a separate step. Honestly most of the time I don't use it at all.

Part of the reason that notifications are so slow to fetch is that it is one of the areas not covered by the graphql api and the rest api has to be used instead. Which is odd, IMO its the area where it would make the most sense to use graphql.

**Reply by artpelling** [reply]
Okay, thanks. I thought there might be a pull-hook or something. Workflows good enough at the moment I guess.
